### DISCUSSION TITLE
Update psalm baseline workflow run

### DISCUSSION THREAD

**Discussion by ry-kumar** [vimeo/psalm#9004] [Q&A]
I got this error while using Psalm for testing ![image](https://user-images.githubusercontent.com/40541080/209455831-8dbef2be-80fe-4a89-a1c7-742b18174a99.jpeg)

**Comment by weirdan** [comment]
Click `View workflow run` and check what was wrong.

**Reply by ry-kumar** [reply]
![image](https://user-images.githubusercontent.com/40541080/209457278-f3adbc06-da3c-4fb0-a099-c78da8f0a2b4.jpeg)

![image](https://user-images.githubusercontent.com/40541080/209457281-2e838abc-cfdd-4d92-b173-75fd0045ad4c.jpeg)

**Reply by orklah** [reply]
Seems like you have a custom stub for gd in you psalm.xml is that correct? It seems be missing
